hi-ml/testhiml/testhiml/test_run_ml.py

```diff
@@ -146,12 +146,12 @@ def test_run_training() -> None:
 
             mock_trainer.fit = Mock()
             mock_close_logger = Mock()
-            mock_trainer.logger = MagicMock(close=mock_close_logger)
+            mock_trainer.loggers = [MagicMock(close=mock_close_logger)]
 
             runner.run_training()
 
             mock_trainer.fit.assert_called_once()
-            mock_trainer.logger.finalize.assert_called_once()
+            mock_trainer.loggers[0].finalize.assert_called_once()
```

Comment:
logger property return first logger why can't we use it anymore?
<img width="454" alt="image" src="https://user-images.githubusercontent.com/37396332/183663861-199e076c-8cc7-4135-8e8b-198eacf4b416.png">

Reply:
We could, I just think this is more explicit and therefore clearer